cmd/ansible-operator/main.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/operator-framework/operator-sdk/pkg/ansible"
+	"github.com/operator-framework/operator-sdk/pkg/ansible/flags"
+	aoflags "github.com/operator-framework/operator-sdk/pkg/ansible/flags"
+	"github.com/operator-framework/operator-sdk/pkg/log/zap"
+)
+
+func main() {
+	logf.SetLogger(zap.Logger())
+
+	flags := setDefaultFlags()
```

Comment:
Why we are re-implementing all flags instead of use the method [AddTo](https://github.com/operator-framework/operator-sdk/blob/master/pkg/ansible/flags/flag.go) as it is done currently for example [here](https://github.com/operator-framework/operator-sdk/blob/cb8bf4744bde5b5d8fe97e35692b497750d24bd3/cmd/operator-sdk/run/cmd.go#L200)?

Reply:
Duplicate the code reduce the Maintenace ability. Note that, if something needs to be changed we need to perform the fix/improvement in more than once place which leads the code more susceptible to errors.

Reply:
Agreed with @camilamacedo86. But I think it would look like this:

```go
func main() {
	flags := aoflags.AddTo(pflag.CommandLine)
	pflag.Parse()

	logf.SetLogger(zap.Logger())

	if err := ansible.Run(flags); err != nil {
		log.Fatal(err)
	}
}
```
Ditto for helm.